config.go

```diff
@@ -796,6 +843,12 @@ func loadConfig() (*config, []string, error) {
 	cfg.ConnectPeers = normalizeAddresses(cfg.ConnectPeers,
 		activeNetParams.DefaultPort)
 
+	// Check the checkpoints for syntax errors.
+	cfg.addCheckpoints, err = parseCheckpoints(cfg.AddCheckpoints)
+	if err != nil {
+		return nil, nil, err
```

Comment:
This will not print anything to the console.  Returning an error just lets btcd know to exist with a non-zero exit code.  You'll need to follow what the other errors in here do:

```Go
if err != nil {
 		err := fmt.Errorf("%s: %s ", funcName, err)
 		fmt.Fprintln(os.Stderr, err)
 		fmt.Fprintln(os.Stderr, usageMessage)
 		return nil, nil, err
```